### PR TITLE
PyGithub Organization Example docs

### PR DESCRIPTION
**Pull request by goodship1**

### CODE REVIEW

File: doc/examples/organization.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (sfdye) [inline doc/examples/organization.rst]: File name not capitalized

File: doc/examples/organization.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Organization
+==========
+
+Get repository's in an organization
+
+.. code-block:: python
+
+	>>>git = Github(access_token)
```
Comment (sfdye) [inline doc/examples/organization.rst]: there should be a space after the `>>>`

File: doc/examples/organization.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+	Repository(full_name="PyGithub/PyGithub")
+
+
+Get all members of an organization. 
+
+
+.. code-block:: python
+	
+	>>>git = Github(access_token)
```
Comment (sfdye) [inline doc/examples/organization.rst]: just use `g` object directly

File: doc/examples/organization.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	>>>org = git.get_organization("PyGithub")
+	>>>membership = org.get_members()
+	>>>for members in membership:
```
Comment (sfdye) [inline doc/examples/organization.rst]: please make sure the style conforms to rest of the examples

**Comment by goodship1** [comment]
cheers i will make the required changes

File: doc/examples/Organization.rst [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Get repository's in an organization
+
+.. code-block:: python
+
+	>>> git = Github(access_token)
+	>>> org = git.get_organization("PyGithub")
+	>>> repository  = org.get_repos()
+	>>> for repos in repository:
+			print(repos)
```
Comment (sfdye) [inline doc/examples/Organization.rst]: It looks like the indentation is off when you preview the change
https://github.com/PyGithub/PyGithub/blob/4ac64d1b21b3aefe20c029003583e1dd0457fe0b/doc/examples/Organization.rst

Comment (goodship1) [reply doc/examples/Organization.rst]: just seen the preview seems different to my copy. I will make the required changes

**Pull request closed without merging**